rust/src/python/helpers.rs

```diff
@@ -76,7 +76,7 @@ impl PartialOrd for DateTimeInfo<'_> {
 pub fn get_tz_name<'py>(dt: &Bound<'py, PyAny>) -> PyResult<String> {
     // let tz: &str = "";
 
-    if !PyDateTime::is_type_of_bound(dt) {
+    if !PyDateTime::is_type_of(dt) {
```

Comment:
Since `dt` is of type `Bound`, shouldn't that be as it was? (I am not yet versed in the pyo3 stuff, so I might be wrong.)

Reply:
Nevermind, I read some more on the matter, and it seems PyO3 is consolidating and removing the `*_bound` methods.